lib/commands/gesture.js

```diff
@@ -27,7 +28,10 @@ commands.click = async function (el) {
 
 commands.clickCurrent = async function (/*button*/) {
   if (this.isWebContext()) {
-    /* TODO */throw new errors.NotYetImplementedError();
+    if (_.isNull(this.curWebCoords)) {
```

Comment:
random comment. I think I prefer `this.curWebCoors === null`, unless `_.isNull` does something I'm not aware of